src/Enforcer.php

```diff
@@ -557,7 +558,6 @@ protected function enforcing(string $matcher, ...$rvals): bool
         $hasEval = Util::hasEval($expString);
 
         if (!$hasEval) {
-            $expressionLanguage = $this->getExpressionLanguage($functions);
             $expression = $expressionLanguage->parse($expString, array_merge($rTokens, $pTokens));
```

Comment:
`$expression`  is initialized as `ExpressionLanguage` at Line 557, could not be converted to string.

Reply:
Yes, Line 557 is redundant. I have deleted